configure.ac

```diff
@@ -101,6 +101,17 @@ AS_IF([test "x$with_pidfile" != xno],
 	])
 AM_CONDITIONAL([WITH_PIDFILE], [test "x$with_pidfile" != xno])
 
+
+AC_ARG_ENABLE([wild],
+[  --enable-wild    Turn on wildcard support],
```

Comment:
A "wild" daemon is probably not a good idea ;)

Maybe the option and description could be changed to more accurately reflect its functionality?